### DISCUSSION TITLE
What is meant by 'waveform data'?

### DISCUSSION THREAD

**Discussion by areapostrema** [chorus-ai/chorus_waveform#9] [Q&A]
Can a definition of 'waveform data' please be provided? What makes data 'waveform'?

**Comment by wa6gz** [comment]
"Waveform data" for CHoRUS purposes are the continuous signals measured from physiological bedside monitors (for example, ECG and raw PPG signals).  Some derived signals, like heart rate from ECG and blood oxygen saturation from PPG, are also calculated and provided by the monitors. Together, the waveforms and derived vital signs/numerics make up the continuous monitoring portion of the CHoRUS data submission.
